src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2627,7 +2627,7 @@ def test_vm_error_on_zone_unavailable(self, resource_group, resource_group_locat
         try:
             self.cmd('vm create -g {rg} -n vm1 --admin-username clitester --admin-password PasswordPassword1! --image debian --zone 1')
         except Exception as ex:
-            self.assertTrue('availablity zone is not yet supported' in str(ex))
```

Comment:
So these changes will require you to bump the VM and ACS modules. You could strip the typo fix and just need to bump ACS.

Reply:
Bumped both ACS and VM modules. This is resolved in 0f83058.